activities.json

```diff
@@ -166,3 +166,31 @@
+{"activity":"Color","accessibility":0.0,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"5322987","enabled":true}
+{"activity":"Organize your pantry","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"3954882","enabled":true}
+{"activity":"Back up important computer files","accessibility":0.2,"type":"busywork","participants":1,"price":0.2,"link":"","key":"9081214","enabled":true}
+{"activity":"Memorize a favourite quote or poem","accessibility":0.8,"type":"education","participants":1,"price":0.0,"link":"","key":"9008639","enabled":true}
+{"activity":"Do something you used to do as a kid","accessibility":0.8,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"8827573","enabled":true}
+{"activity":"Organize your basement","accessibility":0.9,"type":"busywork","participants":1,"price":0.0,"link":"","key":"8203595","enabled":true}
+{"activity":"Write a todo list for the coming weekend","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"9487940","enabled":true}
+{"activity":"Sit in the dark and listen to your favourite music with no distractions","accessibility":1.0,"type":"relaxation","participants":1,"price":0.0,"link":"","key":"9908721","enabled":true}
+{"activity":"Do a jigsaw puzzle","accessibility":1.0,"type":"recreational","participants":1,"price":0.1,"link":"https://en.wikipedia.org/wiki/Jigsaw_puzzle","key":"8550768","enabled":true}
+{"activity":"Look at pictures and videos of cute animals","accessibility":1.0,"type":"relaxation","participants":1,"price":0.0,"link":"","key":"2565076","enabled":true}
+{"activity":"Write a handwritten letter to somebody","accessibility":0.8,"type":"social","participants":1,"price":0.1,"link":"","key":"2277801","enabled":true}
+{"activity":"Match your storage containers with their lids","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"4940907","enabled":true}
+{"activity":"Start a family tree","accessibility":1.0,"type":"social","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Family_tree","key":"6825484","enabled":true}
+{"activity":"Look at your finances and find one way to save money","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"5977626","enabled":true}
+{"activity":"Organize your dresser","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"7556665","enabled":true}
+{"activity":"Learn the periodic table","accessibility":0.6,"type":"education","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Periodic_table","key":"3621244","enabled":true}
+{"activity":"Learn how or improve your touch typing","accessibility":0.8,"type":"busywork","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Touch_typing","key":"2526437","enabled":true}
```

Comment:
Did you mean to say "Learn how to improve your touch typing"?

Reply:
I'll just go with "improve." I was trying to say "Learn if you don't already know or improve if you already do."